build/ci/github_actions/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip docker-hub==2.2.0
+echo yq > dev-requirements.txt
+
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -e operator_image_for_test=$operator_image_for_test\
+                  -e cr_image_value=$cr_image_value\
```

Comment:
`cr_image_value` shouldn't be part of the installation.
could we install it without docker?

Reply:
done